### DISCUSSION TITLE
Run workflow button

### DISCUSSION THREAD

**Discussion by andreaslarssen** [community/community#27049] [Actions]
<p>When clicking the green “Run workflow” button in the pop up box, it does not indicate that the workflow has been initialized.</p>
<p>This can lead to multiple workflows being started, as there’s no way to confirm that the action is processing.</p>
<p>The button should show some kind of indicator that the workflow has been started, or the very least be disabled.</p>
<p>Alternatively, and maybe even better, close the box the button is in as soon as the button is clicked, and add a line in the workflow list that indicates that it has been initialized.</p>

**Comment by lokesh755** [comment, marked as answer]
<p>I just validated that the page refreshes once I clicked the “Run workflow” button which hides the box.</p>

**Comment by andreaslarssen** [comment]
<p>Seems it’s fixed lately. Though, the workflow takes some time to pop into the list, at least it’s better than it was.</p>
